mne/io/brainvision/brainvision.py

```diff
@@ -843,3 +848,204 @@ def _check_bv_annot(descriptions):
     bv_markers = (set(_BV_EVENT_IO_OFFSETS.keys())
                   .union(set(_OTHER_ACCEPTED_MARKERS.keys())))
     return len(markers_basename - bv_markers) == 0
+
+
+def parse_impedance(settings):
```

Comment:
this should be a private function
```suggestion
def _parse_impedance(settings):
```

Reply:
@honzaseda you forgot to make this method private as well